HLTrigger/special/plugins/HLTRegionalEcalResonanceFilter.cc

```diff
@@ -569,7 +566,7 @@ bool HLTRegionalEcalResonanceFilter::filter(edm::Event &iEvent, const edm::Event
     }
   }
 
-  return true;
+  return (eb_collsize > 2 && ee_collsize > 2);
```

Comment:
Is the idea to invert the condition in L558 ? If yes, I guess it should be
```cpp
  return (eb_collsize >= 2 or ee_collsize >= 2);
```
?

Reply:
yep.